lib/index.js

```diff
@@ -121,6 +121,11 @@ export default function slackin({
 
     invite({ token, org, email, channel: chanId }, err => {
       if (err) {
+        if (err.message === `Already invited. Sending you to https://${org}.slack.com...`) {
+          return res
+          .status(403)
+          .json({ msg: err.message, redirectUrl: `https://${org}.slack.com` });
```

Comment:
*(I'm just passing by so ignore me if these comments make no sense)*

Shouldn't it be returning a `301/302` instead of a `403`? Also, why not use `res.redirect("https://${org}.slack.com")`?